frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -78,6 +78,27 @@ const Map: React.FC<MapProps> = ({
   const [baseLayers, setBaseLayers] = useState<BaseLayer[]>([]);
   const [activeBasemap, setActiveBasemap] = useState<BaseLayer | null>(null);
 
+  // different markers for building and parcel
```

Comment:
Can we manually copy these and include them with our build instead of pulling them from the web?

Reply:
+1

Reply:
I agree with this, tried using local paths but currently could not get it working. Will revisit after I get this bug figured out for view/update of buildings

Reply:
did you try using `require` with the icon-url? Agree we can solve this later, but would expect this work...

```tsx
var greenIcon = new Icon({
    iconUrl: require('assets/images/green.svg'),
   ...
});
```

_Assuming you place the icons under `/src/assets/images`_

Reply:
Require was the trick, updating again